package.json

```diff
@@ -4,7 +4,7 @@
   "keywords": [
     "appium"
   ],
-  "version": "0.5.0",
+  "version": "0.6.0",
```

Comment:
Shouldn't need to manually do this. `npm version minor` will update and tag automatically.